app/api/events.py

```diff
@@ -210,15 +278,33 @@ def before_get_object(self, view_kwargs):
                 else:
                     view_kwargs['id'] = None
 
-    decorators = (is_coorganizer, )
+    def after_get(self, result):
+        result = changeToTimezone(result)
```

Comment:
@shubham-padia @niranjan94 
There was an error in the database, now it's working fine.

**Request**
```
{
  "data": {
    "type": "event",
    "attributes": {
    	"name":"First Event",
    	"starts-at":"2017-06-12T12:00:00+05:30",
    	"ends-at":"2017-06-12T12:00:00+05:30",
    	"email":"firstuser@gmail.com"
    }
  }
}
```

**Response :**
```
"starts-at": "2017-06-12T06:30:00+00:00",
"ends-at": "2017-06-12T06:30:00+00:00"
"created-at": "2017-06-12T22:43:42.904375+00:00",
```
And in the **database** :
```
starts_at : 2017-06-12 12:00:00+05:30
ends_at : 2017-06-12 12:00:00+05:30
```